commands/local_server_start.go

```diff
@@ -169,6 +174,17 @@ var localServerStartCmd = &console.Command{
 			}
 		}
 
+		// If 'SSLKEYLOGFILE' environment variable is set, uses this as a destination of TLS key log.
+		// In this context, the name 'SSLKEYLOGFILE' is common, so using 'SSL' instead of 'TLS' name.
+		// The key log file from the console argument or YAML config is preferred than the environment variable.
```

Comment:
I would have expected the env var to take precedence.

Reply:
Okay, I am going to change the behaviour.

Reply:
Changed in [91696a4](https://github.com/symfony-cli/symfony-cli/pull/267/commits/91696a4e4fa5a8aee1b2ea9d8f703fcf10dbb6e8)